build.gradle

```diff
@@ -138,3 +138,9 @@ subprojects {
         testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '3.4.6'
     }
 }
+
+task composeBuild(type: Exec) {
```

Comment:
> running docker-compose -f xxx build now requires that gradle build was called before

the way you have it setup here, won't gradle build just run docker-compose etc... which removes this con that you mentioned?